### PR TITLE
Fix missing pour sounds from reagent dispensers

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request
When pouring liquid out of a barrel or watertank, none of the pour sounds would trigger. This is now fixed. Also added a missing signal and returned `ITEM_INTERACT_SUCCESS` bitflag.

## Why It's Good For The Game
When I play bartender I want consistency.

## Changelog
:cl:
code: Add missing signal to reagent dispenser RMB interactions.
sound: Fix missing pour sounds from reagent dispensers
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/cups/_cup.dm [hunk 1]
```diff
@@ -133,6 +133,7 @@
 			return ITEM_INTERACT_BLOCKING
 
 		var/trans = target.reagents.trans_to(src, amount_per_transfer_from_this, transferred_by = user)
+		playsound(target.loc, pick('sound/effects/liquid_pour1.ogg', 'sound/effects/liquid_pour2.ogg', 'sound/effects/liquid_pour3.ogg'), 50)
```
Comment (Sadboysuss) [inline code/modules/reagents/reagent_containers/cups/_cup.dm:136]: this should really be an sfx define... sigh

File: code/modules/reagents/reagent_containers/cups/_cup.dm [hunk 2]
```diff
@@ -158,9 +159,12 @@
...
```
Comment (Sadboysuss) [inline code/modules/reagents/reagent_containers/cups/_cup.dm:162]: add `vary` to it maybe?

Comment (MrMelbert) [reply code/modules/reagents/reagent_containers/cups/_cup.dm:162]: Clearly it should vary based on the density of the reagents being poured

File: code/modules/reagents/reagent_containers/cups/_cup.dm (hunk 1 above)
Comment (timothymtorres) [reply code/modules/reagents/reagent_containers/cups/_cup.dm]: Sure thing [66f7310](https://github.com/tgstation/tgstation/pull/86859/commits/66f7310466034b252bd8010ab5f493fa9417011f)

File: code/modules/reagents/reagent_containers/cups/_cup.dm (hunk 2 above)
Comment (timothymtorres) [reply code/modules/reagents/reagent_containers/cups/_cup.dm]: Done [66f7310](https://github.com/tgstation/tgstation/pull/86859/commits/66f7310466034b252bd8010ab5f493fa9417011f)

**Review by SmArtKar — APPROVED** [review]
fix the sound path conflicts

**Pull request merged**